account_fiscal_year_closing/models/account_fiscalyear_closing.py

```diff
@@ -90,7 +90,7 @@ def _default_company_id(self):
     closing_template_id = fields.Many2one(
         comodel_name="account.fiscalyear.closing.template",
         string="Closing template",
-        domain="[('chart_template_ids', '=', chart_template_id)]",
```

Comment:
I have no idea and no context... silly question: don't you have to keep the original domain too?

Reply:
Hi @simahawk, with that domain the template has not show

How an you see there:
![image](https://user-images.githubusercontent.com/43987335/149934028-af91cf86-38aa-4874-94ec-7b5577446639.png)

and I have alredy create the template there:
![image](https://user-images.githubusercontent.com/43987335/149934129-74cd7782-dc96-4fec-b20c-49cdc7127b50.png)

Reply:
fine :)